pages/docs/engine/ra-optimisation/search-tree-rao/search-tree-rao.md

```diff
@@ -6,12 +6,125 @@ hide: true
 feature-img: "assets/img/Hans_Otto_Theater_Potsdam_-_fake_colors_cut.jpg"
 tags: [Docs, Search Tree RAO]
 ---
+FARAO search-tree is initially designed to optimize non-costly remedial actions (including generation unit trip 
+where applicable).
 
-### Overview
+Unlike purely linear optimization, a search-tree algorithm does not neglect the non-linearity of efficient remedial 
+actions such as curative and preventive topological actions. Incorrect modeling of remedial actions such as topological 
+remedial actions could lead to over or underestimate their impact and then result in a biased network situation that 
+could endanger the system security.
 
-The search tree RAO module optimizes the Range Actions and topological actions according to a configurable objective function. The solving algorithm deals consecutively with the N state and N-1 states. For each of them, it looks for a good combination of Remedial Actions through:
-- a heuristic search which tests iteratively the impact of the topological actions,
-- linear optimisation steps operated by the [linear RAO module](/docs/engine/ra-optimisation/linear-rao), which optimises the use of PSTs and HVDC lines.
```

Comment:
I think this link was our only way to reach this page

Reply:
added a link in the part i modified to make clearer